documentation/API_documentation/geofencing-subscriptions-API-Readiness-Checklist.md

```diff
@@ -0,0 +1,27 @@
+# API Readiness Checklist
+
+Checklist for geofencing-subscriptions api-version in v0.2
```

Comment:
V0.3 no ?

This morning my understanding is we skip 0.2 for geofencing & location-retrieval to avoid confusion with previous relase that was the 0.2 at family level.